<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>2000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>快速下载器</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFFFFF;</string>
  </property>
  <widget class="QWidget" name="mainContent">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="title" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>11</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>System</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>快速下载器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minBtn">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/fd/images/mini_btn_bg.jpg);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/fd/images/close_btn_bg.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="helpBtn">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>0</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background: transparent;
color:#FFFFFF;</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="toolsBar" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>45</y>
      <width>348</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="Line" name="newRightLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>3</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="newTaskBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>新建任务</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="startAllBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>3</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>开始所有任务</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseAllBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>3</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>暂停所有任务</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="statusBar" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>560</y>
      <width>348</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QCheckBox" name="lockTopCbox">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>12</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>固定</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="downloadList">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>75</y>
      <width>348</width>
      <height>485</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QScrollBar:vertical {
	border:none;
      background:transparent;
      width: 5px;
      margin: 0;
}


QScrollBar:hover {
	width:20px;
}
/**
滚动条拖动按钮
*/
QScrollBar::handle:vertical {
  background:  #46B9F0;
  min-height: 20px;
}
/*
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
  border: 2px solid #00FF00;
  width: 3px;
  height: 3px;
  background: #FF0000;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  background: none;
}*/

/**
滚动条底部按钮块
*/
QScrollBar::add-line:vertical {
  border:none;
  background: transparent;
  height:0px;
  subcontrol-position: bottom;
  subcontrol-origin: margin;
}
/**
滚动条顶部按钮块
*/
QScrollBar::sub-line:vertical {
  border:none;
  background: #FF0000;
  height: 0px;
  subcontrol-position: top;
  subcontrol-origin: margin;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QLabel" name="emptyDownloadImage">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>220</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/fd/images/empty_download_list_img.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
